### PR TITLE
supported platforms file

### PR DESCRIPTION
**Pull request by itsyitsy**
## Proposed changes

Introducing an initial version of supported platform doc for a better overview.

## Types of changes

What types of changes does your code introduce to `thin-edge.io`?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/src/supported-platforms.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# thin-edge.io platform support
+
+This doc`s target is to have an point to come to and find out if thin-edge.io will work on your device.
```
Comment (didier-wenzek) [inline docs/src/supported-platforms.md:3]: I don't think this sentence is helpful. I would simply remove it.

**Review by didier-wenzek — APPROVED** [review]
Approved with an optional improvement proposed.

File: docs/src/supported-platforms.md (hunk 1 above)
Comment (rina23q) [reply docs/src/supported-platforms.md:3]: Also, here you use  `  instead of  '

**Review by rina23q — APPROVED** [review]
LGTM

**Pull request merged**